apps/api/src/domain/domainModelValidators/domainModelValidators.spec.ts

```diff
@@ -49,6 +49,7 @@ const validVocabularyListDTO: PartialDTO<VocabularyList> = {
 const validTagDTO: PartialDTO<Tag> = {
```

Comment:
Test publication filtering

Reply:
I added a story for this [here](https://www.pivotaltracker.com/story/show/181563197).